modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -36,9 +36,6 @@ spec:
         "bridge": "<bridge-interface>", <5>
```

Comment:
I believe we don't need to use the "plugins" list anymore since it is just a single CNI. It should look like this:

```
  config: '{
    "cniVersion": "0.3.1",
    "name": "<a-bridge-network>", (3)
    "type": "cnv-bridge", (4)
    "bridge": "<bridge-interface>", (5)
    "macspoofchk": true, (6)
    "vlan": 1 (7)
  }'
```

Reply:
Fixed